routes/produtos/index.js

```diff
@@ -0,0 +1,213 @@
+
+    // SQL Query > Select Data
+    var query = client.query("SELECT * FROM Produto ORDER BY idProduto DESC LIMIT 1");
+
+    // Stream results back one row at a time
+    query.on('row', function(row) {
+      results.push(row);
+    });
+
+    // After all data is returned, close connection and return results
+    query.on('end', function() {
+      done();
+      return res.json(results);
+    });
+
+
+  });
+}
+
+produtos.update = function(req, res) {
+
+  var results = [];
+
+  // Grab data from the URL parameters
+  var id = req.params.idProduto;
+
+  // Grab data from http request
+  var data = {
+    Valor: req.body.Valor,
+    Nome: req.body.Nome,
+    Imagem: req.body.Imagem,
+    Descrição: req.body.Descrição,
+    Peso: req.body.Peso,
+    Tamanho: req.body.Tamanho,
+    Fabricante: req.body.Fabricante,
+    Quantidade: req.body.Quantidade,
+    Tipo: req.body.Tipo
+  };
+
+  // Get a Postgres client from the connection pool
+  pg.connect(connectionString, function(err, client, done) {
+    // Handle connection errors
+    if(err) {
+      done();
+      console.log(err);
+      return res.status(500).send(json({ success: false, data: err}));
+    }
+
+    // SQL Query > Update Data
+    client.query("UPDATE Produto SET Valor=($1), Nome=($2), Imagem=($3), Descrição=($4), Peso=($5), Tamanho=($6), Fabricante=($7), Quantidade=($8), Tipo=($9) WHERE idProduto=($10)", [data.Valor, data.Nome, data.Imagem, data.Descrição, data.Peso, data.Tamanho, data.Fabricante, data.Quantidade, data.Tipo, id]);
+
+    // SQL Query > Select Data
+    var query = client.query("SELECT * FROM Produto ORDER BY idProduto ASC");
+
+    // Stream results back one row at a time
+    query.on('row', function(row) {
+        results.push(row);
+    });
+
+    // After all data is returned, close connection and return results
+    query.on('end', function() {
+      done();
+      return res.json(results);
+    });
+  });
+
+}
+
+produtos.delete = function(req, res) {
+
+  var results = [];
+
+  // Grab data from the URL parameters
+  var id = req.params.idProduto;
+
+
+  // Get a Postgres client from the connection pool
+  pg.connect(connectionString, function(err, client, done) {
+    // Handle connection errors
+    if(err) {
+      done();
+      console.log(err);
+      return res.status(500).json({ success: false, data: err});
+    }
+
+    // SQL Query > Delete Data
+    client.query("DELETE FROM Produto WHERE idProduto=($1)", [id]);
+
+    // SQL Query > Select Data
+    var query = client.query("SELECT * FROM Produto ORDER BY idProduto ASC");
+
+    // Stream results back one row at a time
+    query.on('row', function(row) {
+      results.push(row);
+    });
+
+    // After all data is returned, close connection and return results
+    query.on('end', function() {
+      done();
+      return res.json(results);
+    });
+  });
+
+}
+
+module.exports = produtos;
```

Comment:
I wonder if it's too big for a single file?
Maybe we should split into post/get/update/delete files?